build/.goreleaser.yml

```diff
@@ -21,6 +23,8 @@ builds:
     binary: nri-nginx
     ldflags:
       - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    env:
+      - CGO_ENABLED=0
     goos:
       - windows
     goarch:
```

Comment:
Should we add as well the change:

```
RUN git config --global --add safe.directory '*'
```
?

Reply:
I do not see it [here](https://github.com/newrelic/nri-nginx/blob/master/build/Dockerfile)